SecureSDLCNotes.md

```diff
@@ -162,7 +162,14 @@ Monitoring is key to detect incidents and respond to them in a timely manner. It
 
 [NIST SP 800-30][15]
 
+[Separation of duties. Wikipedia][4]
+
+[Secure Product Design. OWASP][5]
+
 [2]:https://snyk.io/learn/application-security/static-application-security-testing/ (Static Application Security Testing. Snyk)
 [1]: https://learn.microsoft.com/en-us/azure/security/develop/threat-modeling-tool-threats#stride-model (Threat Modelling Tool Threats, Microsoft)
 [14]:https://owasp.org/www-community/OWASP_Risk_Rating_Methodology (OWASP Risk Rating Methodology, OWASP)
 [15]:https://csrc.nist.gov/pubs/sp/800/30/r1/final (NIST SP 800-30 Rev. 1, NIST)
+[4]: https://en.wikipedia.org/wiki/Separation_of_duties (Wikipedia)
```

Comment:
Wikipedia regularmente no es tan buena referencia, y en el caso de este artículo hace referencia tanto a contabilidad como a IT.
A lo mejor podrías tomar una de las referencias dentro de Wikipedia mejor agregarla a este documento.